src/style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  text-align: center;
+  background-color: rgb(218, 214, 214);
+  height: 100vh;
+}
+
+.flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.rm-decoration {
+  list-style: none;
+  text-decoration: none;
+}
+
+ul {
+  padding: 30px;
+  background-color: white;
+  width: 40%;
+  text-align: left;
+}
```

Comment:
Good job working on the structure of the Todo list and dynamically fetching and displaying the todos from the array of todos.

- [ ] It would be nice if you could style your todo app similar to the one given in the project requirements
- [ ] Please work on your CSS styles to match the design given

**The Original Design**
![image](https://user-images.githubusercontent.com/50941074/178855684-1f7b6edc-bc7c-4534-841c-ae43766cfb14.png)

**Your design**
![image](https://user-images.githubusercontent.com/50941074/178855850-404eedcb-b88e-42be-b570-2d39fea67c84.png)